ui/src/shared/components/threesizer/Division.tsx

```diff
@@ -123,6 +126,53 @@ class Division extends PureComponent<Props> {
     }
   }
 
+  private get renderDragHandle(): JSX.Element {
+    const {draggable} = this.props
+
+    return (
+      <div
+        style={this.handleStyle}
+        title={this.title}
+        draggable={draggable}
+        onDragStart={this.drag}
+        className={this.handleClass}
+        onDoubleClick={this.handleDoubleClick}
+      >
+        {this.renderDragHandleContents}
+      </div>
+    )
+  }
+
+  private get renderDragHandleContents(): JSX.Element {
+    const {
+      name,
+      handlePixels,
+      orientation,
+      headerButtons,
+      menuOptions,
+    } = this.props
+
+    if (!name) {
+      return
+    }
+
+    if (orientation === HANDLE_HORIZONTAL && handlePixels >= 20) {
```

Comment:
this `handlePixels >= 20` is used in more than once. how do you feel about making that 20 a constant?

Reply:
Done!